core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -480,8 +481,7 @@ oppia.directive('conversationSkin', [
                 nextSupplementalCardIsNonempty) {
               PlayerPositionService.setActiveCardIndex(numVisibleCards - 1);
               animateToTwoCards(function() {});
-            } else if (
-                totalNumCards > 1 &&
+            } else if (totalNumCards > 1 &&
```

Comment:
> I have moved totalNumCards to the previous line as in the if block above.

No -- I think I mentioned that if the contents of (...) span more than one line, we should break after '('?

Reply:
Sorry -- I saw this review comment just now. Making changes!

Reply:
Done -- thanks!